tests/integration/components/file-details/dynamic-scan/manual-test.js

```
@@ -1440,5 +1541,180 @@ module(
+            dynamic_status: ENUMS.DYNAMIC_STATUS.INQUEUE,
+          });
+
+          return new Response(201);
+        }
+      );
+
+      this.server.put('/profiles/:id/device_preference', (schema, req) => {
+        const data = req.requestBody
+          .split('&')
+          .map((it) => it.split('='))
+          .reduce((acc, [key, value]) => ({ ...acc, [key]: value }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)